src/thmap.c

```diff
@@ -406,7 +431,11 @@ get_leaf(const thmap_t *thmap, thmap_inode_t *parent, unsigned slot)
 {
 	thmap_ptr_t node;
 
-	node = atomic_load_explicit(&parent->slots[slot], memory_order_relaxed);
+	/*
+	 * Consume operation pairs with release fence or release
+	 * operation in thmap_put.
```

Comment:
Can we come up with some notation to keep these comments more succinct? It also pairs with thmap_del(). That's why I have half-hearted feeling towards such annotations.. they are verbose and often just boil down to "readers synchronize with writers" and vice-versa. Hardly a discovery.